models/weatherSchema.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
+
+const weatherSchema = mongoose.Schema({
+  minTem : String,
+  maxTem : String, 
```

Comment:
I wonder why these two fields are declaired as string. Can you give me some explanation?